bin/create-gitops-namespace-files.rb

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "fileutils"
+
+# TODO: default
+# TODO: question N of M
+# TODO: validate answers
+# TODO: check namespace name is available
+# TODO: remove old terraform stuff from namespace-resources/
+# TODO: mount user's home directory into the tools image, so we can write deployment files to their working copy of their app.
+
+TEMPLATES_DIR = "gitops-namespace-resources"
+DEPLOY_TEMPLATES_DIR = "#{TEMPLATES_DIR}/kubectl_deploy"
+CLUSTER_NAME = "gitops-test.cloud-platform.service.justice.gov.uk"
```

Comment:
This will need to be live-1 for master. Thinking about it, should we set this variable in the pipeline?

Reply:
Fixed.